<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrudCareer</class>
 <widget class="QMainWindow" name="CrudCareer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>306</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>306</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>684</width>
    <height>306</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Administración de carreras</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/book-flat.ico</normaloff>:/book-flat.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>651</width>
      <height>261</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Raleway</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tbInsert">
     <attribute name="title">
      <string>Agregar carrera</string>
     </attribute>
     <widget class="QLabel" name="lbIdCareer">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>92</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Id carrera</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCareer">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>122</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Carrera</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitle">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(35, 54, 86);</string>
      </property>
      <property name="text">
       <string>Administración de carreras</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneIdCareer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneCareer">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>55</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pbInsert">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pbInsert {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbInsert:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>Insertar carrera</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbExit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>130</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pbExit {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbExit:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
      </property>
      <property name="text">
       <string>Salir al menú principal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbQuery">
     <attribute name="title">
      <string>Buscar carrera</string>
     </attribute>
     <widget class="QLabel" name="lbSearch">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>14</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Busqueda:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneCritery">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>12</y>
        <width>321</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cboCritery">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>12</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvQuery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>621</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tbUpdate">
     <attribute name="title">
      <string>Modificar carrera</string>
     </attribute>
     <widget class="QGroupBox" name="gbIdCareer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLineEdit" name="lneIdQuery">
       <property name="geometry">
        <rect>
         <x>77</x>
         <y>13</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="lbIdQuery">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>15</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Id carrera</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSearch">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>121</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbSearch {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbSearch:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="gbCareer">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>621</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="lbCareerQuery">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>67</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Carrera</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lneCareerQuery">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>65</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>55</number>
       </property>
      </widget>
      <widget class="QPushButton" name="pbExit2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>80</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbExit2 {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbExit2:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Salir al menú principal</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbUpdate">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>10</y>
         <width>291</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbUpdate {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbUpdate:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Actualizar carrera</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-16</x>
      <y>-5</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 91, 118);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit2</sender>
   <signal>clicked()</signal>
   <receiver>CrudCareer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
